client/src/components/FriendsPage/FriendsPage.js

```diff
@@ -0,0 +1,137 @@
+import axios from "axios";
+import React from "react";
+
+import { withStyles } from "@material-ui/core/styles";
+import MuiPaper from "@material-ui/core/Paper";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+
+import { theme } from "../../themes/theme";
+import FriendsListSkeleton from "../Skeletons/FriendsListSkeleton";
+import TabPanel from "./TabPanel";
+import TabPanelContent from "./TabPanelContent";
+
+const categories = ["suggestions", "followers", "followings"];
+const apis = ["/friends", "/friends/followers", "/friends/followings"];
+
+const Paper = withStyles({
+  root: {
+    width: "100%",
+    maxWidth: 500,
+    height: "100%",
+  },
+})(MuiPaper);
+
+export default function FriendsPage() {
+  const [selectedTab, setSelectedTab] = React.useState(0);
+  const [loading, setLoading] = React.useState(true);
+  const [fetchedFriends, setFetchedFriends] = React.useState(
+    new Array(categories.length).fill([])
```

Comment:
It's not a huge deal but are these default values necessary? It seems like fetchedFriends (and maybe keywords) can just be defaulted to `[]` and still work.

Reply:
Ah, that's right. I'm gonna change that👍 
(Might not change keywords... )